src/Compilers/CSharp/Portable/Symbols/AllowsRefLikeTypeSymbolVisitor.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed class AllowsRefLikeTypeSymbolVisitor : SymbolVisitor
```

Comment:
Are there any other cases than just `params T[]` that need to be handled specially?

- If yes, we should have tests for these cases.
- If no, we don't need a visitor, we could simply check just that one case.

Reply:
Not as far as I know